project-base/storefront/pages/_error.tsx

```diff
@@ -48,13 +48,8 @@ ErrorPage.getInitialProps = getServerSidePropsWrapper(({ redisClient, domainConf
     //    Boundary. Read more about what types of exceptions are caught by Error
     //    Boundaries: https://reactjs.org/docs/error-boundaries.html
 
-    if (context.err) {
-        logException(context.err);
-    } else {
-        // If this point is reached, getInitialProps was called without any
-        // information about what the error might be. This is unexpected and may
-        // indicate a bug introduced in Next.js, so record it in Sentry
-        logException(new Error(`_error.js getInitialProps missing data at path: ${context.asPath}`));
+    if (context.res.statusCode !== 404) {
+        logException(context.err || new Error(`_error.js getInitialProps missing data at path: ${context.asPath}`));
     }
```

Comment:
I am not sure if it was already decided, but have you thought about removing logging to Sentry on this page? As we talked about it. So just to be sure this was handled.